src/main/java/com/cloudbees/groovy/cps/tool/Driver.java

```diff
@@ -78,18 +78,21 @@ public void run() throws Exception {
              *      2. has a Closure parameter in one of the arguments, not in the receiver
              */
             Predicate<ExecutableElement> selector = (e) -> {
-                return e.getKind() == ElementKind.METHOD
-                    && e.getModifiers().containsAll(PUBLIC_STATIC)
-                    && e.getParameters().subList(1, e.getParameters().size()).stream()
-                        .anyMatch(p -> t.types.isAssignable(p.asType(), closureType))
-                    && !EXCLUSIONS.contains(e.getEnclosingElement().getSimpleName().toString()+"."+e.getSimpleName().toString());
+                boolean r = e.getKind() == ElementKind.METHOD
+                        && e.getModifiers().containsAll(PUBLIC_STATIC)
+                        && e.getParameters().subList(1, e.getParameters().size()).stream()
+                                .anyMatch(p -> t.types.isAssignable(p.asType(), closureType))
+                        && e.getAnnotation(Deprecated.class) == null
```

Comment:
Skipping some deprecated methods whose implementations wind up producing ambiguous overloads. Though perhaps these would be solved by the problem currently blocking me on `each`…